src/services/Email.js

```diff
@@ -0,0 +1,252 @@
+      clientEmail: client_email,
+      clientPhone: client_phone,
+      clientId: client_id,
+      url: adminUrl
+    },
+  };
+
+  try {
+    await sendEmail(message);
+  } catch (error) {
+    const message = {
+      to: 'supporst@helloroo.zendesk.com',
+      from: 'no-reply@helloroo.org',
+      subject: `Client request help email notification error - ${Date.now()}`,
+      templateId: 'd-3c6b31897e0540418d2a202aa396b357',
+      substitutions: {
+        clientFirstName: client_first_name,
+        clientLastName: client_last_name,
+        clientEmail: client_email,
+        coachEmail: process.env.PM_EMAIL,
+        error: error.response
+      }
+    }
+    await sendgridClient.send(message);
+  }
+};
+
+/**
+ * This function receives clients details and sends email to the Coach
+ * if the client has sent a stop message
+ * 
+ * @param {Object} userData - userData contains clients details.
+ * 
+ * @returns {void}
+ */
+const emailCoachOnClientStop = async (userData) => {
+  /* eslint-disable */
+  const {
+    coach_email,
+    coach_name,
+    client_email,
+    client_first_name,
+    client_last_name,
+    client_phone,
+    client_plan_url
+  } = userData;
+
+  const message = {
+    to: coach_email,
+    from: 'support@helloroo.org',
+    subject: 'Stop message from client',
+    templateId: '34bf6299-14fd-4e8d-b019-2bdda54a0fd1',
+    substitutions: {
+      clientFirstName: client_first_name,
+      clientLastName: client_last_name,
+      clientEmail: client_email,
+      clientPhone: client_phone,
+      clientPlanUrl: client_plan_url,
+      coachFirstName: coach_name
+    },
+  };
+
+  try {
+    await sendEmail(message);
+  } catch (error) {
+    const message = {
+      to: 'support@helloroo.zendesk.com',
+      from: 'no-reply@helloroo.org',
+      subject: `Stop message email notification error - ${Date.now()}`,
+      templateId: 'd-3c6b31897e0540418d2a202aa396b357',
+      substitutions: {
+        clientFirstName: client_first_name,
+        clientLastName: client_last_name,
+        coachEmail: coach_email,
+        error: error.response.body.errors[0].message
+      }
+    }
+    await sendgridClient.send(message);
+  }
+};
+
+
+/**
+ * This function is responsible for sending email.
+ * 
+ * @param {Object} message - Receives message as a parameter and send email using Sendgrid.
+ * 
+ * @returns {void}
+ */
+
+const sendEmail = async (message) => {
+  try {
+    await sendgridClient.setApiKey(process.env.SENDGRID_API_KEY);
+    await sendgridClient.send(message);
+  } catch (error) {
+    const message = {
+      to: 'supporst@helloroo.zendesk.com`',
+      from: 'no-reply@helloroo.org',
+      subject: `Failure to send Email- ${Date.now()}`,
+      templateId: 'd-3c6b31897e0540418d2a202aa396b357',
+      substitutions: {
+        error: error.response.body.errors[0].message
+      }
+    }
+    await sendgridClient.send(message);
+  }
+}
+
+
+exports.sendCoachEmail = sendCoachEmail;
```

Comment:
Hey Ricky, why is `exports.module` reassigned to `exports.sendCoachEmail`? Ditto for the next 3 lines.

Reply:
@65black 
Because all those functions are part of the `exports` object. 
That way, they can be shared/imported from other locations.